gum/gumeventsink.c

```diff
@@ -60,3 +79,46 @@ gum_event_sink_stop (GumEventSink * self)
   if (iface->stop != NULL)
     iface->stop (self);
 }
+
+static void
+gum_default_event_sink_class_init (GumDefaultEventSinkClass * klass)
+{
+  GObjectClass * object_class = G_OBJECT_CLASS (klass);
+  object_class->finalize = gum_default_event_sink_finalize;
+}
+
+static void
+gum_default_event_sink_iface_init (gpointer g_iface, gpointer iface_data)
+{
+  GumEventSinkInterface * iface = g_iface;
+  iface->query_mask = gum_default_event_sink_query_mask;
+  iface->process = gum_default_event_sink_process;
+}
+
+static void
+gum_default_event_sink_init (GumDefaultEventSink * self)
+{
+}
+
+static void
+gum_default_event_sink_finalize (GObject * object)
+{
+  G_OBJECT_CLASS (gum_default_event_sink_parent_class)->finalize (object);
+}
```

Comment:
If a `dispose` or `finalize` implementation does nothing but chain up, it should be omitted.